tests/lib/log.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
IMO just use the `logging` module. It's well understood and you can manipulate it directly in the main entry point of the test suite.